### PR TITLE
Change provider to supplier

### PR DESCRIPTION
**Pull request by momodonatien**
DONE

### CODE REVIEW

File: src/KitBox/Database/ProviderManager.cs [hunk 1]
```diff
@@ -3,92 +3,92 @@
...
+	public SupplierManager(NpgsqlConnection conn)
 	{
 		this.connection = conn;
 	}
 
-	public Provider SelectProvider(string nameSociety)
+	public Supplier SelectSupplier(string nameSociety)
 	{
-		Provider provider = null;
+        Supplier supplier = null;
```
Comment (azerupi) [inline src/KitBox/Database/ProviderManager.cs]: Indentation

File: src/KitBox/Database/ProviderManager.cs [hunk 2]
```diff
@@ -3,92 +3,92 @@
...
 		this.command = new NpgsqlCommand(select, this.connection);
 
 		this.command.Parameters.Add(new NpgsqlParameter("name_society", NpgsqlDbType.Varchar)).Value = nameSociety;
 
 		NpgsqlDataReader reader = this.command.ExecuteReader();
 		while (reader.Read())
 		{
-			provider = new Provider((string)reader["name_society"], (string)reader["name_shop"], (string)reader["address"], (string)reader["city"]);
-			provider.Id = (int)reader["id"];
+            supplier = new Supplier((string)reader["name_society"], (string)reader["name_shop"], (string)reader["address"], (string)reader["city"]);
```
Comment (azerupi) [inline src/KitBox/Database/ProviderManager.cs]: indentation

File: src/KitBox/Database/ProviderManager.cs [hunk 3]
```diff
@@ -3,92 +3,92 @@
...
+		this.command.Parameters.Add(new NpgsqlParameter("name_shop", NpgsqlDbType.Varchar)).Value = supplier.Shop;
+		this.command.Parameters.Add(new NpgsqlParameter("address", NpgsqlDbType.Varchar)).Value = supplier.Address;
+		this.command.Parameters.Add(new NpgsqlParameter("city", NpgsqlDbType.Varchar)).Value = supplier.City;
 
 		this.command.ExecuteNonQuery();
 
 		this.connection.Close();
 
-		provider.Id = SelectProvider(provider.Society).Id;
+        supplier.Id = SelectSupplier(supplier.Society).Id;
```
Comment (azerupi) [inline src/KitBox/Database/ProviderManager.cs]: indentation

File: src/KitBox/Database/ProviderManager.cs [hunk 4]
```diff
@@ -3,92 +3,92 @@
...
 		this.command = new NpgsqlCommand(select, this.connection);
 		NpgsqlDataReader reader = this.command.ExecuteReader();
-		List<Provider> providers = new List<Provider>();
+		List<Supplier> suppliers = new List<Supplier>();
 		while (reader.Read())
 		{
-			Provider provider = new Provider((string)reader["name_society"], (string)reader["name_shop"], (string)reader["address"], (string)reader["city"]);
-			provider.Id = (int)reader["id"];
-			providers.Add(provider);
+            Supplier supplier = new Supplier((string)reader["name_society"], (string)reader["name_shop"], (string)reader["address"], (string)reader["city"]);
```
Comment (azerupi) [inline src/KitBox/Database/ProviderManager.cs]: indentation

File: src/KitBox/Database/Company.cs [hunk 5]
```diff
@@ -57,9 +57,9 @@ public InvoiceManager InvoiceManager
 		get { return this.invoiceManager; }
 	}
 
-	public ProviderManager ProviderManager
-	{
-		get { return this.providerManager; }
+	public SupplierManager SupplierManager
+    {
```
Comment (azerupi) [inline src/KitBox/Database/Company.cs]: indentation

**Comment by azerupi** [comment]
There are a couple of indentation mismatches. Either you conform to the existing style or you format the whole file (you should be able to use auto-formatting).

**Comment by azerupi** [comment]
The changes look good, though I would prefer if the person merging could test it beforehand. I don't expect any trouble, but just to be sure.

File: src/KitBox/Database/ProviderManager.cs [hunk 6]
```diff
@@ -3,92 +3,92 @@
...
+            supplier.Id = (int)reader["id"];
 		}
 		reader.Close();
 		this.connection.Close();
-		return provider;
+		return supplier;
 	}
 
-	public Provider InsertProvider(Provider provider)
+	public supplier InsertSupplier(Supplier supplier)
```
Comment (bluebeel) [inline src/KitBox/Database/ProviderManager.cs]: public Supplier et non public supplier

**Pull request merged**